monai/transforms/signal/array.py

```diff
@@ -0,0 +1,773 @@
+        """
+        Args:
+            signal: signal to be normalized
+        """
+        if len(signal.shape) == 1:
+            signal = np.expand_dims(signal, axis=0)
+
+        signal = (signal - np.min(signal)) / (np.max(signal) - np.min(signal))
+        return signal
+
+
+class SignalStandardize(Transform):
+    """
+    Standardize a signal
+    """
+
+    backend = [TransformBackends.NUMPY]
+
+    def __call__(self, signal: np.ndarray) -> np.ndarray:
+        """
+        Args:
+            signal: signal to be standardized
+        """
+        if len(signal.shape) == 1:
+            signal = np.expand_dims(signal, axis=0)
+
+        signal = (signal - signal.mean()) / signal.std()
+        return signal
+
+
+class SignalZeroPad(Transform):
+    """
+    Zero pad a signal
+    """
+
+    backend = [TransformBackends.NUMPY]
+
+    def __init__(self, targetlength: int = 2000, *args, **kwargs) -> None:
+        """
+        Args:
+            targetlength: target length of the signal
+        """
+        super().__init__()
+        self.targetlength = targetlength
+
+    def __call__(self, signal: np.ndarray) -> np.ndarray:
+        """
+        Args:
+            signal: signal to be zero padded
+        """
+
+        if len(signal.shape) == 1:
+            signal = np.expand_dims(signal, axis=0)
+
+        extended_signal = np.zeros([signal.shape[0], self.targetlength])
+        siglength = np.min([self.targetlength, signal.shape[1]])
+        extended_signal[:, :siglength] = signal[:, :siglength]
+
+        return extended_signal
+
+
+class SignalFillEmpty(Transform):
+    """
+    replace empty part of a signal
+    """
+
+    backend = [TransformBackends.NUMPY]
+
+    def __init__(self, replacement: float = 0.0, *args, **kwargs) -> None:
+        """
+        Args:
+            replacement: value to replace nan items in signal
+        """
+        super().__init__()
+        self.replacement = replacement
+
+    def __call__(self, signal: np.ndarray) -> np.ndarray:
+        """
+        Args:
+            signal: signal to be filled
+        """
+
+        return np.nan_to_num(signal, nan=self.replacement)
+
+
+class SignalRemoveFrequency(Transform):
+    """
+    Remove a frequency from a signal
+    """
+
+    backend = [TransformBackends.NUMPY]
+
+    def __init__(
+        self, frequency: float = 60, quality_factor: float = 15, sampling_freq: float = 500, *args, **kwargs
+    ) -> None:
+        """
+        Args:
+            frequency: frequency to be removed from the signal
+            quality_factor: quality factor for notch filter
+            see : https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.iirnotch.html
+            sampling_freq: sampling frequency of the input signal
+        """
+        super().__init__()
+        self.frequency = frequency
+        self.quality_factor = quality_factor
+        self.sampling_freq = sampling_freq
+
+    def __call__(self, signal: Any) -> Any:
```

Comment:
`Any` is used here, but `np.ndarray` is used elsewhere.

Reply:
fixed